visualize-admin/visualization-tool
# feat: Disable transitions when resizing dashboard charts vertically

Until now, we only disabled transitions when width was changing; we should also disable it when height changes, which solves the issues with vertically resizing charts in free canvas dashboard layout.

## How to test
**PR**
1. Go to [this link](https://visualization-tool-git-feat-dashboard-improvements-ixt1.vercel.app/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/9&dataSource=Prod&flag__layouter.dashboard.free-canvas=true).
2. Add a second chart.
3. Proceed to layout options.
4. Resize any of the charts vertically.
5. ✅ See that the chart resizing is immediate.

**TEST**
1. Go to [this link](https://test.visualize.admin.ch/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/9&dataSource=Prod&flag__layouter.dashboard.free-canvas=true).
2. Add a second chart.
3. Proceed to layout options.
4. Resize any of the charts vertically.
5. ❌ See that the chart resizing is lagging.
